packages/client/utils/date/relativeDate.ts

```diff
@@ -19,6 +21,45 @@ interface Opts {
   smallDiff?: string
 }
 
+/**
+ * Creates a human-readable string representing the time till the given date,
+ * for example: 2 days left; 13 hours left; 2 days left, etc or null if the date is in the past
+ * @param date
+ */
+export const humanReadableCountdown = (date: string | Date) => {
```

Comment:
+2 we should consider using this for timers in other meetings that are more than a few minutes!

Reply:
good idea, will create a task for it